experimental/PropertyDDS/packages/property-inspector-table/src/HashCalculator.ts

```diff
@@ -240,7 +240,7 @@ export class HashCalculator {
 
     // Compute the hash for the remaining bytes in the buffer
     const buffer = this.hashBuffer.Uint8HashArray;
-    /* eslint-disable no-fallthrough, no-multi-spaces */ // Allows cases to fallthrough without complaints.
+    /* eslint-disable no-multi-spaces */ // Allows cases to fallthrough without complaints.
```

Comment:
nit: I don't think the trailing comment here is valid any more since we've removed the no-fallthrough disable.

Reply:
Correct. Fixed.